backend/clickhouse/traces.go

```diff
@@ -79,6 +84,7 @@ var defaultTraceKeys = []*modelInputs.QueryKey{
 	{Name: string(modelInputs.ReservedTraceKeyDuration), Type: modelInputs.KeyTypeNumeric},
 	{Name: string(modelInputs.ReservedTraceKeyParentSpanID), Type: modelInputs.KeyTypeString},
 	{Name: string(modelInputs.ReservedTraceKeySecureSessionID), Type: modelInputs.KeyTypeString},
+	{Name: string(modelInputs.ReservedTraceKeyMetricName), Type: modelInputs.KeyTypeString},
```

Comment:
I wasn't able to add metric value on the frontend so curious how you were able to add, or if we should add here?

Reply:
ReservedTraceKeyMetricValue is here already, should match the METRIC_VALUE_COLUMN on the frontend.

Reply:
![Screenshot from 2024-06-25 15-20-04](https://github.com/highlight/highlight/assets/1351531/b90bf2b9-3c26-47d8-a285-4ed7daae1354)